pkg/flavor/kubernetes/flavor.go

```diff
@@ -49,10 +50,11 @@ type Flavor struct {
 	claimStopChan chan struct{}
 
 	eventRecorder record.EventRecorder
+	nfsMonitor    *Monitor
 }
 
 // NewKubernetesFlavor creates a new k8s flavored CSI driver
-func NewKubernetesFlavor(nodeService bool) (*Flavor, error) {
+func NewKubernetesFlavor(nodeService bool, nfsMonitor bool, nfsMonitorInterval int64) (*Flavor, error) {
```

Comment:
It might be cleaner to just do this in a separate function:

func ConfigureMonitor(type nfs|minio, monitorInterval)

Thus allowing the client to determine which type and interval to apply.